### PR TITLE
DIP-9: Devcon 5.1

### PR DESCRIPTION
**Pull request by crisgarner**

### CODE REVIEW

File: DIPs/DIP-6.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+DIP: 6
```
Comment (ligi) [inline DIPs/DIP-6.md:2]:
```suggestion
DIP: 9
```

**Review by ligi — CHANGES_REQUESTED** [review]
Thanks for the PR - unfortunately DIP-6 was submitted before (the ticketing PR from alphawallet) - so please rename to DIP-9.md and change the DIP in the header.
otherwise LGTM

**Comment by crisgarner** [comment]
@ligi Thanks! updated to 9

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
@crisgarner thanks! LGTM to me now - after a second editor will look over it we should be good to merge

**Comment by ligi** [comment]
sorry for the delay - we are still young in the process and currently some people are in holiday. Going through the new checklist (#22) now:

 * [x] Correct markdown (e.g. headers with ##, header surrounded by ---)
 * [x] syntax check
 * [ ] filename is DIP-X.md (with X being the correct DIP number)
 * [ ] header contains (in this order)
   * [ ] DIP
   * [ ] Title
   * [ ] Theme
   * [ ] Tags
   * [ ] Author
   * [ ] Resources Required
   * [ ] Discussion-to (with link to devcon forum)
   * [ ] Created
 * [ ] contains these sections:
   * [ ] Abtract
   * [ ] Rationale
   * [ ] Implementation
   * [ ] Operational Reuirements & Ownership
 * [ ] PR title prefixed with DIP-X

**Pull request merged**